x-pack/plugin/inference/src/yamlRestTest/resources/rest-api-spec/test/inference/20_semantic_text_field_mapper.yml

```diff
@@ -55,25 +56,7 @@ setup:
           index: test-index
           id: doc_1
           body:
-            non_inference_field: "you know, for testing"
```

Comment:
These tests were meant to address the parsing of the inference results. I don't see this explicitly tested in other tests. Should we keep these?

Reply:
I moved these tests to InferenceMetadataFieldMapperTests since it is not possible anymore to provide the _inference field manually in a bulk request.